class-vip-support-user.php

```diff
@@ -746,10 +771,11 @@ public static function is_verified_automattician( $user_id ) {
 
 		$instance = self::init();
 
-		$is_a8c_email 	= $instance->is_a8c_email( $user->user_email );
-		$email_verified = $instance->user_has_verified_email( $user->ID );
+		$is_a8c_email 	  = $instance->is_a8c_email( $user->user_email );
+		$is_allowed_email = $instance->is_allowed_email( $user->user_email );
```

Comment:
Minor, but might be worth renaming the function to `is_verified_automattician_or_allowed_user()` or  to avoid confusion.

Reply:
Alternatively, we could refactor it out to a different function like `is_allowed_support_user()` in the future case where we do want to restrict access to only a verified automattician and not an allowed user. Of course, that would require updating the code in vip-go-mu-plugins, but could be worth it.

Reply:
Also, ^^ this is all under the assumption that non-a8c users will be defined in `VIP_SUPPORT_USER_ALLOWED_EMAILS`.

Reply:
Also, ^^ this is all under the assumption that non-a8c users will be defined in `VIP_SUPPORT_USER_ALLOWED_EMAILS`. If that's not intended, might be worth checking if it's an a8c somewhere. Thoughts, @WPprodigy?